Packs/Okta/Integrations/Okta_v2/Okta_v2.yml

```
@@ -1326,6 +1326,14 @@ script:
     - description: 'Update Proxy IP addresses: CIDR range (1.1.0.0/16) or single IP address (2.2.2.2).'
       isArray: true
       name: proxyIPs
+    - description: 'Indicate the action of adding an IP to an existing zone or overriding the existing IPs.'
```

Not sure this description is clear
```suggestion
    - description: 'Indicate the update type: APPEND - add an IP to an existing list of zone IPs. OVERRIDE: overriding the existing IP list with the given IP.'
```